packages/web-components/@sfx/products/README.md

```diff
@@ -0,0 +1,33 @@
+# SF-X Products Component
+
+## Functionality
+The `sfx-products` component acts as a wrapper around a series of individual
+`sfx-product` components. It uses a list of information about products
+and passes each item to a single product for rendering.
+
+### Received Events
+
+#### `sfx::provide-products`
+Upon receiving this event, the `sfx-products` component will accept
+the passed products data and update/add/remove any child product tiles.
+
+## Customization
+The SF-X Products component allows for configuring the quantity of displayed
+products. The following attributes are available:
+- `maxitems`: A number that limits the quantity of product tiles shown.
```

Comment:
This should be removed now that the component no longer accepts this attribute. The sentence should really just be something like, "this component has no customization options."